channel-fuel-testnet.toml

```diff
@@ -39,58 +39,58 @@ url = "https://github.com/FuelLabs/forc-explorer/releases/download/v0.28.1/forc-
 hash = "9ef1d8af2c5a611d33f8966881197bd4958da05e4b325eb0dc847b84ea1eac96"
 
 [pkg.forc-wallet]
-version = "0.7.1"
+version = "0.8.1"
 
 [pkg.forc-wallet.target.aarch64-unknown-linux-gnu]
-url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.7.1/forc-wallet-0.7.1-aarch64-unknown-linux-gnu.tar.gz"
-hash = "61eb3cca081402ace5e663808ad929550250d4b639ba871f8ab59991b6032008"
+url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.8.1/forc-wallet-0.8.1-aarch64-unknown-linux-gnu.tar.gz"
+hash = "b1b2d36ad7c19bc239d00ca78aa8adad18cb82f1b41e97393e7a0179c368c3fb"
 
 [pkg.forc-wallet.target.x86_64-unknown-linux-gnu]
-url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.7.1/forc-wallet-0.7.1-x86_64-unknown-linux-gnu.tar.gz"
-hash = "b62da85962b9beff29b5d0051011e311dcda6809c953fa09da9672ec69edaad2"
+url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.8.1/forc-wallet-0.8.1-x86_64-unknown-linux-gnu.tar.gz"
+hash = "0d36f5ec2c5911a393db47f5764862c5f14f16dcaa26d9668a752f536baddac6"
 
 [pkg.forc-wallet.target.aarch64-apple-darwin]
-url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.7.1/forc-wallet-0.7.1-aarch64-apple-darwin.tar.gz"
-hash = "475a7a78cd607ab4d0f92f3d5ae0a7a6d8c52d2722d215ba31ffc367ad78d290"
+url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.8.1/forc-wallet-0.8.1-aarch64-apple-darwin.tar.gz"
+hash = "85c44424c36cc354148c902079bba5666a10fdb53cccb04545a0b9606190fa6f"
 
 [pkg.forc-wallet.target.x86_64-apple-darwin]
-url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.7.1/forc-wallet-0.7.1-x86_64-apple-darwin.tar.gz"
-hash = "001155c56d78e4cebf381ee3a250dfbf62c73b857da0f385abc74a21bb0ea8a6"
+url = "https://github.com/FuelLabs/forc-wallet/releases/download/v0.8.1/forc-wallet-0.8.1-x86_64-apple-darwin.tar.gz"
+hash = "5f6e0de39fe684bf08fb666a18322a5927de823efed02b36707b7368c67626c2"
 
 [pkg.fuel-core]
-version = "0.26.0"
+version = "0.28.0"
```

Comment:
Shouldn't this point to `fuel-core` @ `0.30.0`?

![testnmet](https://github.com/FuelLabs/fuelup/assets/26660/8f93252e-0925-4daa-aec0-89073473c5d2)

Reply:
The sway repo is still on 0.28.0